js102/index.html

```diff
@@ -55,21 +55,78 @@
 					<p>Slides: <a href="http://yonet.github.io/JS102/#/" target="_blank">bit.ly/JS102</a></p>
 				</section>
 				<!-- /Welcome -->
+
+				<!-- Tone Setting -->
+				<section>
+					<h2>Hello!</h2>
+					<p>Some 'guidelines' for today</p>
+					<ul>
+						<li>We are here to support your learning in a safe, inclusive environment</li>
```

Comment:
THANK YOU for great feedback and catching those syntax mishaps. Let's catch
up soon!

On Sun, Mar 6, 2016 at 10:30 AM Brenda Jin <notifications@github.com> wrote:

> In js102/index.html
> <https://github.com/gdisf/teaching-materials/pull/198#discussion_r55145571>
> :
>
> > @@ -55,21 +55,78 @@
> >  					<p>Slides: <a href="http://yonet.github.io/JS102/#/" target="_blank">bit.ly/JS102</a></p>
> >  				</section>
> >  				<!-- /Welcome -->
> > +
> > +				<!-- Tone Setting -->
> > +				<section>
> > +					<h2>Hello!</h2>
> > +					<p>Some 'guidelines' for today</p>
> > +					<ul>
> > +						<li>We are here to support your learning in a safe, inclusive environment</li>
>
> I really like these guidelines and wonder if we should update them in the
> intro slides
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdisf/teaching-materials/pull/198/files#r55145571>.
>
-- 
Jaclyn Karnowski

freelance web developer
co-founder, EdTalks